.github/workflows/long-tests.yaml

```diff
@@ -10,9 +14,20 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  placeholder:
-    name: Placeholder job
+  js-long:
+    name: Long JetStream tests
     runs-on: ${{ vars.GHA_WORKER_SMALL }}
     steps:
-      - name: Dummy step
-        run: echo "Hello World"
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          # Use go version from go.mod
+          go-version-file: "go.mod"
```

Comment:
@mprimi 
In go.mod we specify the lowest supported go version (1.22.0) at the moment, but we actually build releases with the version specified in travis:
https://github.com/nats-io/nats-server/blob/5e6017135b4b1d333b435ff55b720275a39bb7af/.travis.yml#L64-L67
(1.23.5 at the moment)
So this test runs with go toolchain version that is not used for the release
https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Was this an intention here?